python/main.py

```diff
@@ -43,3 +76,14 @@ async def get_image(image_name):
         image = images / "default.jpg"
 
     return FileResponse(image)
+
+@app.get("/items/{item_id}")
+def get_item(item_id: int):
+    with open(json_file, mode='r') as j:
+        items = json.load(j)
+
+    if item_id >= len(items['items']):
```

Comment:
Nice job, but you should also consider if the item id is less than 0 or if the item id is not a number at all.